src/Sylius/Behat/Page/Admin/ProductVariant/IndexPageInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\ProductVariant;
+
+use Sylius\Behat\Page\Admin\Crud\IndexPageInterface as BaseIndexPageInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+interface IndexPageInterface extends BaseIndexPageInterface
+{
+    /**
+     * @param ProductVariantInterface $productVariant
+     * @param int $quantity
+     *
+     * @return bool
+     */
+    public function hasOnHandQuantityFor(ProductVariantInterface $productVariant, $quantity);
+
+    /**
+     * @param ProductVariantInterface $productVariant
+     * @param int $quantity
+     *
+     * @return bool
+     */
+    public function hasOnHoldQuantityFor(ProductVariantInterface $productVariant, $quantity);
+
+    /**
+     * @param ProductVariantInterface $productVariant
+     *
+     * @return bool
+     */
+    public function hasOnHoldLabelFor(ProductVariantInterface $productVariant);
```

Comment:
This method should be removed.

What do you think about refactoring `hasOn*QuantityFor(ProductVariantInterface $productVariant, $quantity) : bool` to `getOn*QuantityFor(ProductVariantInterface $productVariant) : int` and comparing it in the context?

This way we can also show nicer exceptions.